documentation/reference/qfieldcloud/system.en.md

```diff
@@ -4,102 +4,107 @@ tx_slug: documentation_reference_qfieldcloud_system
+| Remove from project         | Remove the layer from the project                                                              | Remove the layer from the project | Remove the layer from the project                                                                |
+
+In summary, with QFieldCloud:
+
+- *Offline editing* means that a offline copy of the data will be generated by QFieldCloud and downloaded to QField.
+The data from *Offline editing*-layers will all be stored as multiple layers within a single `data.gpkg`.
+Whenever a modification is made on QField, a JSON structure called a *Change* (or Delta) will be generated only for the features that have been modified, only for the attributes or the geometry that have been modified.
+On push, the original data source will be updated, including the database for database layers.
+- *Directly access data source* is mainly used for service-based layers that are located on a internet accessible server.
+The data in these layers are directly modified by QField.
+Examples are WFS, WM(T)S-layers or layers coming from a database such as PostGIS layers.
+If the layer is file-based it will be in read-only mode.
+- *Remove from project* will simply remove the layer from the project (not package it for QField).
+
+From QFieldSync it will be possible to update a project already loaded on QFieldCloud.
+
+In the event that the changes concern only styles, forms etc. but not the data or the structure of the layers, the QGIS project file (`.qgs`/`.qgz`) on the server will simply be updated.
+
+If there are changes in the data or the structure of the layers, then the new data source files will be updated.
+
+Note that changing the structure (or schema) of the layers such as adding, removing or renaming attributes, changing not null constraints, etc. might have effects.
+If such changes are pushed to QFieldCloud, please make sure all QField users have pushed their changes in advance.
+Otherwise it might be impossible for QFieldCloud to apply these Changes/Deltas from older version of the QGIS project.
+
+
+#### Behaviour of QField
+
+Depending on the actions set for each layer in QFieldSync for QFieldCloud or Cable export, QField will act as follows:
+
+| Action                          | File based layer          | Service-based layer (e.g. WMS, database)   | Notes                                |
+|---------------------------------|---------------------------|--------------------------------------------|--------------------------------------|
+| Offline editing                 | Create and push a Change  | N/A                                        |                                      |
+| Directly access data source     | Layer is readonly         | Edit the online online database            |                                      |
```

Comment:
Edit the online database

(there are two "onlines"

Reply:
```suggestion
| Directly access data source     | Layer is readonly         | Edit the online  database                  |                                      |
```